libs/coin-modules/coin-tron/src/network/index.ts

```diff
@@ -360,7 +363,10 @@ const getTrc20 = async (
 
   return {
     results: transactions.data,
-    nextUrl: transactions.meta.links?.next,
+    nextUrl: transactions.meta.links?.next?.replace(
```

Comment:
![image](https://github.com/LedgerHQ/ledger-live/assets/19901996/e87adc2c-9eb5-4ec6-ae6f-16088b7cf522)
![image](https://github.com/LedgerHQ/ledger-live/assets/19901996/1ce5206b-b61d-49b7-9153-897aa8cdd312)

Maybe we can just use the fingerprint parameter for each next page ?
There is no fingerprint parameter when reaching the last one